src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Common/SyntaxNodeExtensions.cs

```diff
@@ -173,44 +197,74 @@ public static bool HasUsingStatement(this SyntaxNode node, string namespaceName)
                 .Concat(children.OfType<VBSyntax.SimpleImportsClauseSyntax>().Select(i => i.Name.ToString()))
                 .Concat(children.OfType<VBSyntax.ImportsStatementSyntax>().SelectMany(i => i.ImportsClauses.OfType<VBSyntax.SimpleImportsClauseSyntax>().Select(i => i.Name.ToString())));
 
-            return usings.Any(n => n.Equals(namespaceName, StringComparison.Ordinal));
+            return usings.Any(n => n.Equals(namespaceName, node.Language == LanguageNames.CSharp ? StringComparison.Ordinal : StringComparison.OrdinalIgnoreCase));
```

Comment:
nice, always appreciate when someone reminds me to be case insensitive for VB

Reply:
I believe we have an extension method to get a comparison based on the node....

Reply:
It looks like we have an extension for symbols but not syntax. I'll add one and use it.